repo/fsrepo/defaultds.go

```diff
@@ -31,16 +31,8 @@ func openDefaultDatastore(r *FSRepo) (repo.Datastore, error) {
 		return nil, fmt.Errorf("unable to open leveldb datastore: %v", err)
 	}
 
-	// 4TB of 256kB objects ~=17M objects, splitting that 256-way
-	// leads to ~66k objects per dir, splitting 256*256-way leads to
-	// only 256.
-	//
-	// The keys seen by the block store have predictable prefixes,
-	// including "/" from datastore.Key and 2 bytes from multihash. To
-	// reach a uniform 256-way split, we need approximately 4 bytes of
-	// prefix.
 	syncfs := !r.config.Datastore.NoSync
-	blocksDS, err := flatfs.New(path.Join(r.path, flatfsDirectory), 4, syncfs)
+	blocksDS, err := flatfs.New(path.Join(r.path, flatfsDirectory), 7, syncfs)
```

Comment:
@whyrusleeping I am not 100% sure, but I think this value should be 6 to preserve the original intent.  Here is why, the initial '/' should not be base32 encoded so the original 4 bytes will now be
```
1 + 3*8/5 = 5.8 = 6 bytes
```
instead of
```
4*8/5 = 6.4 = 7 bytes
```